ilastik/widgets/collapsibleWidget.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         widget: QWidget,
         header: str = "Details",
         animation_duration: int = 100,
+        init_expanded=False,
```

Comment:
Maybe drop the `init_` prefix? Since it's a widget, it is assumed that the expanded state could change later.